ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+##Create counter variable
+counter = 0
+
+##ask user for number of employees
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+while counter < employees  
+  #ask user input and convert to correct variable types
+  puts "What is you name?"
+  identifier = gets.chomp.to_s
+
+  puts "How old are you?"
+  age = gets.chomp.to_i
+
+  puts "What year were you born?"
+  year = gets.chomp.to_i
+
+  ##checks user input for garlic aversion and store the value as a boolean
+  puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+  if gets.chomp.to_s == "yes"
+    garlic_preference = true
+  else
+    garlic_preference = false
+  end
+
+  #checks input for health insurance and stores as a boolean
+  puts "Would you like to enroll in the company’s health insurance?"
+  if gets.chomp.to_s == "yes"
+    insurance = true
+  else
+    insurance = false
+  end
+
+  ##begin until loop asking the user for allergies
+  allergy_input = nil ##sets initial value of user input
+  until allergy_input == "sunshine" || allergy_input == "done"
+    puts "Please name any allergies one at a time, type done when complete"
+    allergy_input = gets.chomp.to_s
+    if allergy_input == "sunshine"
+      vampire = true
+    else
+      vampire = false
+    end
+  end
+
+  ##calculate age from birth year given
+  calculated_age = Time.new.year - year
+
+  ##test age match and store value
+  if calculated_age == age
+    age_match = true
+  else
+    age_match = false
+  end
+
+  ##verify if user is vampire using if blocks and logic expressions
```

Comment:
The tough part of this exercise is needing to test all of the specs and return the value of the last test, since an if/else tree will stop looking once it hits its first match, consider writing many smaller if/else statements instead and updating a variable. Then you can output what you want at the end based on the value of the variable.